packages/sit-onyx/src/components/OnyxListbox/OnyxMiniSearch.vue

```diff
@@ -0,0 +1,101 @@
+<script setup lang="ts">
+import { computed, useAttrs, type HtmlHTMLAttributes } from "vue";
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+import xSmall from "@sit-onyx/icons/x-small.svg?raw";
+import { injectI18n } from "../../i18n";
+
+defineOptions({ inheritAttrs: false });
+
+const attrs = useAttrs();
+
+const props = defineProps<{ modelValue: string; label: string }>();
```

Comment:
Should we align with other components and create a `types.ts` file with the props?

Reply:
Then I'd at least move it to a dedicated type and export it from the .vue file so it can be easily used :)

Reply:
done